docs/source/algorithms/ParallaxCorrection-v1.rst

```diff
@@ -0,0 +1,54 @@
+
+.. algorithm::
+
+.. summary::
+
+.. relatedalgorithms::
+
+.. properties::
+
+Description
+-----------
+
+This algorithm performs a geometrical correction for the so-called parallax effect in tube based SANS instruments.
+
+The correction formula must be specified in the :ref:`IPF <InstrumentParameterFile>` as follows:
+
+- A string parameter named **direction** must hold **x** or **y** which is the direction of the tubes in the detector.
+
+- A string parameter named **parallax** must hold the `muparser <http://beltoforion.de/article.php?a=muparser>`_ expression, where **t** is reserved for the parallax angle:
+
+:math:`t = \arctan(\frac{x}{z})` if direction is **y**, and :math:`t = \arctan(\frac{y}{z})` if direction is **x**.
+
+:math:`x, y, z` are the coordinates of the detector pixel in the system where sample is at :math:`0,0,0` and :math:`z` is the beam axis.
+:math:`t \in (0,\frac{\pi}{2})` in radians.
+
+The correction will be calculated for each pixel and the input data will be divided by the correction and stored in the output.
+
+The instrument parameters must be defined for detector components and without loss of generality, different components can have different formulae.
+
+At least one component name must be given as input.
```

Comment:
I will add an XML example. Regarding the magnitude of the correction, it depends of course on the formula. So I think what you are suggesting would be more relevant in the reduction technique documentation, rather than here in the algorithm.

Reply:
Sure, but perhaps if there is a simple test case you've been using then you could do a screenshot of that? What about a screen shot of the instrument view for the corrected workspace? If you really don't think it is worth it then do not worry.